.circleci/config.yml

```diff
@@ -172,70 +126,43 @@ jobs:
 workflows:
   version: 2
 
-  # The main workflow responsible for triggering all other workflows
-  # in which changes are detected.
-  ci:
-    when: << pipeline.parameters.trigger >>
-    jobs:
-      - trigger-workflows
-
-  # Trigger tag build when tag and not already building a specific package workflow
-  tag:
-    when:
-      and:
-        - << pipeline.git.tag >>
-        - not: << pipeline.parameters.wpe-headless-plugin >>
-    jobs:
-      - trigger-tag:
-          filters:
-            tags:
-              only: /^plugin\/wpe-headless\/\S+/
-
   # Workflows defined for each package.
-  # By default circleci doesn't trigger on a tag event. Add the filter for tags so that all jobs in the workflow will run also when a tag is present.
+  # tag example for deploying an update for wpe-headless: plugin/wpe-headless/1.0.0
   plugin-wpe-headless:
-    when: << pipeline.parameters.wpe-headless-plugin >>
     jobs:
-      - checkout:
-          filters:
-            tags:
-              only: /.*/
+      - checkout
       - plugin-wpe-headless-checkout:
           requires:
             - checkout
-          filters:
-            tags:
-              only: /.*/
       - plugin-wpe-headless-composer:
           requires:
             - plugin-wpe-headless-checkout
-          filters:
-            tags:
-              only: /.*/
       - plugin-wpe-headless-bundle-zip:
           requires:
             - plugin-wpe-headless-checkout
           filters:
+            branches:
+              ignore: /.*/
             tags:
-              only: /.*/
+              only: /^plugin\/wpe-headless\/\S+/
       - plugin-wpe-headless-bundle-json:
           requires:
             - plugin-wpe-headless-checkout
           filters:
+            branches:
+              ignore: /.*/
             tags:
-              only: /.*/
+              only: /^plugin\/wpe-headless\/\S+/
       - plugin-wpe-headless-deploy:
           name: "Deploy zip to api (staging)"
           requires:
             - plugin-wpe-headless-bundle-zip
             - plugin-wpe-headless-bundle-json
           filters:
             branches:
-              only:
```

Comment:
This was here to only push/[ublish/deploy the zip file on merge, not every competing pr and branch constantly pushing zip files of different flavors to the staging endpoint.  That way staging represented at least what was in develop in 'canary'.